src/torrent_peer_allocator.cpp

```diff
@@ -129,8 +129,8 @@ namespace libtorrent {
             return;
         }
 #endif
-		TORRENT_ASSERT(m_ipv4_peer_pool.is_from(static_cast<libtorrent::ipv4_peer*>(p)));
-		static_cast<libtorrent::ipv4_peer*>(p)->~ipv4_peer();
+		TORRENT_ASSERT(m_ipv4_peer_pool.is_from(static_cast<libtorrent::aux::ipv4_peer*>(p)));
+		static_cast<libtorrent::aux::ipv4_peer*>(p)->~ipv4_peer();
```

Comment:
same here, I would expect you could remove `libtorrent::` instead

Reply:
I will do it